src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdTypeDataProvider.java

```diff
@@ -9,101 +9,11 @@
   private AdTypeDataProvider() {
   }
 
-  @DataProvider
-  public static final Object[][] asyncSuccessWithAd() {
```

Comment:
What do you think of testing this ad type in another slot for example MIC or new BLB?

Reply:
It is tested with (m)BLB, only `DataProvider` is eliminated
https://github.com/Wikia/selenium-tests/pull/2195/files#diff-788533f3604f2f7912f879dcea9e412fR24

Reply:
OK, we've got it: https://github.com/Wikia/selenium-tests/pull/2195/files#diff-788533f3604f2f7912f879dcea9e412fR24